20200326/AGENDA.md

```diff
@@ -0,0 +1,28 @@
+# Sardana Follow-up Meeting - 2020/03/26
+
+To be held on 2020/03/26 at 14:00
+
+## Agenda
+
+1. Urgent user problems/issues - Round table
+2. Review pending points from the previous meeting
+    - [ ] What about SOLARIS involvement in the Sardana project?
```

Comment:
I think there were more TODO actions identified in the last meeting:
- [ ]  Zibi will review the last debug information from Thorsten.
- [ ]  Juliano will share the code of the proxy/mirror controller that they use
- [ ] 

Basically all which start with a check box.